server/services/echo_service.go

```diff
@@ -46,21 +47,26 @@ func (s *echoServerImpl) Echo(ctx context.Context, in *pb.EchoRequest) (*pb.Echo
 	if err != nil {
 		return nil, err
 	}
+	echoHeaders(ctx)
 	echoTrailers(ctx)
 	return &pb.EchoResponse{Content: in.GetContent(), Severity: in.GetSeverity()}, nil
 }
 
 func (s *echoServerImpl) Expand(in *pb.ExpandRequest, stream pb.Echo_ExpandServer) error {
 	for _, word := range strings.Fields(in.GetContent()) {
+		log.Printf("range_stream: %v:", stream)
 		err := stream.Send(&pb.EchoResponse{Content: word})
 		if err != nil {
+			log.Printf("err_stream: %v:", stream)
 			return err
 		}
 	}
+	echoStreamingHeaders(stream)
 	if in.GetError() != nil {
 		return status.ErrorProto(in.GetError())
 	}
 	echoStreamingTrailers(stream)
+	log.Printf("ending_stream: %v:", stream)
```

Comment:
Remove this too

Reply:
Done! Thanks again for catching and reviewing.